test/src/test/java/org/corfudb/runtime/clients/ManagementClientTest.java

```diff
@@ -72,6 +74,12 @@ public void handleBootstrap()
                 .isInstanceOf(ExecutionException.class);
     }
 
+    @Test
+    public void addNodeWorkflowRPCTest() throws Exception {
```

Comment:
Add a description of what this test does

Reply:
Ok.